### DISCUSSION TITLE
How to submit plugin for display on Hexo.io website?

### DISCUSSION THREAD

**Discussion by lorezyra** [hexojs/hexo#5174] [Question / Answer] [answered]
I built a [new plugin](https://www.npmjs.com/package/hexo-generator-readtime) recently and published it to `NpmJS.com`. It doesn't show up on the [Hexo plugins](https://hexo.io/plugins/) page. 

Is there a procedure to get it listed there?

## Related
- https://github.com/hexojs/hexo/discussions/5169

**Comment by yoshinorin** [comment, marked as answer]
@lorezyra 
Please edit [`source/_data/plugins.yml`](https://github.com/hexojs/site/blob/2f4a821d641f65681292d533cfdabb17821776d1/source/_data/plugins.yml) in https://github.com/hexojs/site and submit PR.

> see: https://hexo.io/docs/plugins#Publishing

Thank you :)
